pkg/apis/serving/v1beta1/explainer.go

```diff
@@ -35,6 +35,9 @@ type ExplainerSpec struct {
 	PodSpec `json:",inline"`
 	// Component extension defines the deployment configurations for explainer
 	ComponentExtensionSpec `json:",inline"`
+
+	// Spec for multiple storage uris.
+	StorageUris []StorageUri `json:"storageUris,omitempty"`
```

Comment:
May we also consider also making the StorageSpec support multiple models?

Reply:
Agreed, storage spec was originally introduced mainly for model mesh and I think we should consider reducing the complexity for the two different ways of specifying storage.